CMakeLists.txt

```diff
@@ -73,7 +69,11 @@ macro(targets)
   )
 endmacro()
 
-call_for_each_rmw_implementation(targets GENERATE_DEFAULT)
+# For whatever reason when I enable this along with Connext, it fails to
+# link.  Disable for now.
+#call_for_each_rmw_implementation(targets GENERATE_DEFAULT)
+
+targets()
```

Comment:
But there is no `target_suffix` variable anymore. Therefore those should be removed to avoid any confusion.